pkg/common/keys.go

```diff
@@ -194,4 +194,10 @@ const (
 
 	// ArgoCDDeletionFinalizer is a finalizer to implement pre-delete hooks
 	ArgoCDDeletionFinalizer = "argoproj.io/finalizer"
+
+	// ArgoCDDefaultServer is the default server address
+	ArgoCDDefaultServer = "https://kubernetes.default.svc"
+
+	// ArgoCDClusterSecretLabel is needed for cluster secrets
+	ArgoCDClusterSecretLabel = "argocd.argoproj.io/secret-type"
```

Comment:
I think this constant should be named `ArgoCDSecretTypeLabel` instead (it will be used for more than just `cluster` secrets).

Reply:
done !